tests/FSharp.Analyzers.Tests/FSharp.Analyzers.Tests.fsproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
Is this necessary?

Reply:
FSharp.Analyzers.SDK.Testing appears to be .NET 7.0+ now, so I think it needs to be updated to 7 or 8 (just picked 8 as 7 is out of support now)

Reply:
Yep, sounds reasonable. I wasn't sure if that change got reverted or not.